client/my-sites/plans/jetpack-plans/iterations.ts

```diff
@@ -42,7 +43,17 @@ const getCurrentCROIterationName = (): Iterations | null => {
 	}
 
 	const showSocialProofHeader = abtest( 'jetpackSocialProofHeader' ) === 'withSocialProof_test';
-	return showSocialProofHeader ? Iterations.SPROOF : null;
+	const showSecurityProducts = abtest( 'pricingPagePopularProducts' ) === 'withSecurityRT_test';
+
+	if ( showSocialProofHeader ) {
```

Comment:
We'll rebase this PR with trunk once we have a winner for the ongoing experiment.